go.mod

```diff
@@ -73,6 +73,8 @@ require (
 	lukechampine.com/blake3 v1.1.7
 )
 
+replace github.com/lightninglabs/neutrino v0.14.2 => /Users/marci/git/neutrino
```

Comment:
OK to proceeding with an update to the required to point to 792af8548c14a112dafba06f4d8081436d9ec8ca
It's just your commit past the v0.14.2 tag.